graspologic/plot/plot.py

```diff
@@ -671,6 +676,242 @@ def pairplot(
+    X : array-like, shape (n_samples, n_features)
+        Input data.
+    j : int
+        column index of feature of interest from X which will be the x-axis data.
+    k : int
+        column index of feature of interest from X which will be the y-axis data.
+    means : array-like, shape (n_components, n_features)
+        Estimated means from gmm
+    covariances : array-like, shape (with 'full') (n_components, n_features, n_features)
+        estimated covariances from gmm
+    ax : axis object
+        The location where plot_ellipse will plot
+    label_palette : dict, optional, default: dictionary using 'Set1'
+    cluster_palette : dict, optional, default: dictionary using 'Set1'
+    alpha : float, optional, default: 0.7
+        Opacity value of plotter markers between 0 and 1
+
+    References
+    ----------
+    .. [1] https://matplotlib.org/3.1.1/gallery/shapes_and_collections/ellipse_demo.html#references
+    """
+    sns.scatterplot(
+        data=data, x=X[:, j], y=X[:, k], ax=ax, hue="labels", palette=label_palette
+    )
+    for i, (mean, covar) in enumerate(zip(means, covariances)):
+        v, w = linalg.eigh(covar)
+        v = 2.0 * np.sqrt(2.0) * np.sqrt(v)
+        u = w[0] / linalg.norm(w[0])
+        # Plot an ellipse to show the Gaussian component
+        angle = np.arctan(u[1] / u[0])
+        angle = 180.0 * angle / np.pi
+        ell = mpl.patches.Ellipse(
+            [mean[j], mean[k]],
+            v[0],
+            v[1],
+            180.0 + angle,
+            color=cluster_palette[i],
+        )
+        ell.set_clip_box(ax.bbox)
+        ell.set_alpha(alpha)
+        ax.add_artist(ell)
+        # removes tick marks from off diagonal graphs 
+        ax.set_xticks([]), ax.set_yticks([])
+        ax.set_ylabel(k), ax.set_xlabel(k)
+        ax.legend().remove()
+
+
+def pairplot_with_gmm(
+    X,
+    gmm,
+    labels=None,
+    cluster_palette=None,
+    label_palette=None,
+    title=None,
+    context="talk",
+    font_scale=1,
+    alpha=0.7,
+    figsize=(12, 12),
+    kde=True,
+):
+    r"""
+    Plot pairwise relationships in a dataset.
+    By default, this function will create a grid of Axes such that each
+    dimension in data will by shared in the y-axis across a single row and
+    in the x-axis across a single column.
+
+    Parameters
+    ----------
+    X : np.ndarray (2D)
+        input graph
+    gmm: Gaussian Mixture Model object
+    labels : 1d np.ndarray or list, same length as dimensions of X
+        Labels for different categories of graph nodes
+        If labels are not passed in then labels assume the same value as
+        the gmm clusters each point as
+    label_palette : str or dict, optional,
+        with no input : dictionary using 'Set1'
+    cluster_palette : str or dict, optional, default: dictionary using 'Set1'
+        with no input : dictionary using 'Set1'
+    title : string
+        default : "Fit a Gaussian mixture using " + n_components + " components"
```

Comment:
need to update this as well

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/fd0c77041c284d103be7cc9ed57b754072fe092f